File: app/src/main/java/com/neome/api/meta/base/dto/DefnDtoVideoTimestamp.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdVideoTimestamp;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnDtoVideoTimestamp
{
  @Nullable
  public MetaIdVideoTimestamp metaId;

  @Nullable
  public Long startTimeMinutes;

  @Nullable
  public Long startTimeSeconds;

  @Nullable
  public String style;

  @Nullable
  public String title;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnDtoVideoTimestamp.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdVideoTimestamp

interface DefnDtoVideoTimestamp
{
  val metaId: MetaIdVideoTimestamp?
  val startTimeMinutes: Long?
  val startTimeSeconds: Long?
  val style: String?
  val title: String?
}